Sources/RestBird/Request/UploadRequest.swift

```diff
@@ -11,13 +11,20 @@ public enum UploadSource {
     case url(URL)
     case data(Data)
     case stream(InputStream)
```

Comment:
Since we know which mimeTypes are supported here maybe will be better to define an enum for the mime types

Reply:
Maybe this approach is better :) https://github.com/halcyonmobile/RestBird/pull/21#discussion_r